semgrep/Pipfile

```diff
@@ -11,4 +11,6 @@ pytest-xdist = "*"
 tox = "*"
 
 [packages]
+click = "~=8.0.1"
```

Comment:
These need to be put in setup.py for packaging

Reply:
Ah, in bento we pulled this in automatically. I'll investigate doing that here.